Makefile

```diff
@@ -184,6 +184,20 @@ lib:
 	cd lib && \
 	$(MAKE)
 
+# Runs a test of ./lib
+libtest: FORCE
+	rm -rf install_target
+	mkdir -p install_target
+	cd lib && \
+	 $(MAKE) clean && \
+	 $(MAKE) all && \
+	 $(MAKE) install PREFIX=../install_target
+	test -d install_target/include
+	test -f install_target/include/khmer.hh
+	test -d install_target/lib
+	test -f install_target/lib/libkhmer.a
+	rm -rf install_target
```

Comment:
Should your `tst.cc` be included here to catch issues similar to the Seqan header problem in the future?